exercises/exercise3.js

```diff
@@ -8,5 +8,145 @@ module.exports.botScripts = [
     handler: function () {
       return 'Good Evening to You!';
     },
+    //I want the bot to find the to be have a varity of moods when asked a question.
+
+    label: 'Whats your mood bot',
+    prompt: 'Next are you happy',
+    handler: function () {
+      return 'Yes i am';
+    }, //this will tell me what my name is
+    label: 'I want to make a discord bot',
+    prompt: 'What is my #{name}',
+    handler: function (params) {
+      const customer = params.name;
+      return 'Your name is ' + params.name;
+    },
+    isCaseSensitive: false,
+    isListening: true,
+  },
+  {
+    //This will return the users status if he is online or not
+    label: 'Whats my status',
+    prompt: 'Am offline or online?',
+    handler: function () {
+      const onlineStatus = 20;
+      const offlineStatus = 75;
+      const num = 50;
+      if (onlineStatus < 50) {
+        return 'You are online!';
+      } else if (offlineStatus > 50) {
+        return 'You are offline!';
```

Comment:
You may already know this, but this code will always return 'You are online!' because the first 'if' statement is currently hardcoded as true and a return was provided (which returns itself right out of the function ;)) You could use Math.random() here to create a random number to change the online or offline status at random, but just a suggestion to make it more dynamic.